support/cas-server-support-hazelcast-core/src/main/java/org/apereo/cas/hz/HazelcastConfigurationFactory.java

```diff
@@ -126,6 +126,12 @@ public static Config build(final BaseHazelcastProperties hz) {
             networkConfig.setPublicAddress(cluster.getPublicAddress());
         }
 
+        if (!org.springframework.util.CollectionUtils.isEmpty(cluster.getOutboundPorts())) {
+            for (val port : cluster.getOutboundPorts()) {
```

Comment:
I suspect you can also use a lambda syntax here instead of the for-loop.

Reply:
and yes